### PR TITLE
add a simple workflow

### PR DESCRIPTION
**Pull request by SaulLu**
Added a workflow to test on push that : 
- the dependencies are correct
- the code format is correct

### CODE REVIEW

**Comment by SaulLu** [comment]
I don't know how to do this in the most efficient way because I think I've made the necessary changes to be compatible with the Flake8 format in the next PR that is coming up (logging_2 branch).

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: fhir2dataset
+
+on: [push]
+
+jobs:
+  build:
```
Comment (simonvadee) [inline .github/workflows/main.yml:6]: don't you want to add a job to publish on pypi ? there is an example on pyfhirstore

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    name: Lint and test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ '3.7', '3.8']
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
```
Comment (Jasopaum) [inline .github/workflows/main.yml:13]: You can also remove that comment 🙂

File: .github/workflows/main.yml (hunk 1 above)
Comment (SaulLu) [reply .github/workflows/main.yml:6]: Done ! :smile:

File: .github/workflows/main.yml (hunk 2 above)
Comment (SaulLu) [reply .github/workflows/main.yml]: Done :smiley:

**Pull request closed without merging**